src/index.js

```diff
@@ -1,11 +1,3 @@
-let a = prompt('Enter your name');
```

Comment:
No, try leave old logic(only logic, not code) and move to variables duplicated strings